src/track/text-track-display.js

```diff
@@ -792,7 +792,12 @@ class BoxPosition {
       case '+y':
         return this.top < container.top;
       case '-y':
-        return this.bottom > container.bottom;
+        // IE has too many digits after the decimal point which results in wrong calculation.
+        try{
```

Comment:
should be applied to all switch case options, so better move this to helper method and apply to all cases.